### ISSUE TITLE
Realtime Active Thread Chart not found problem

### ISSUE THREAD

**Issue by zfaster** [naver/pinpoint#4570]
I deploy the same agent to the two web server,the real time chart one is normal but another was show not found. how can I solve this problem
![image](https://user-images.githubusercontent.com/8341790/44887577-856dbe80-acff-11e8-865e-b651e6f785c0.png)

**Comment by koo-taejin** [comment]
hi @zfaster 

Please check the status of the agent at that time.
If do not have a problem, Please post related collector log and web log at that time. 

thanks.

**Comment by zfaster** [comment]
@koo-taejin thank you reply.
![image](https://user-images.githubusercontent.com/8341790/44890122-86a4e880-ad0b-11e8-95cd-be768e30935c.png)
the agent is normal and can receive data

this is web and collector log,I can not find error due to this problem

[web.txt](https://github.com/naver/pinpoint/files/2338602/web.txt)

[collector.txt](https://github.com/naver/pinpoint/files/2338608/collector.txt)

**Comment by zfaster** [comment]
@koo-taejin 
the machine have two ip a Internal and public ,do you know why the collector connect to web use public ip?
I guess is web register to zookeeper use public ip , how can i change the ip to Internal ?

**Comment by Shengairong** [comment]
I got the same issue. On the Thread Dump page, it displays:
----------------------------------
NOT_FOUND 
For the above reasons, this agent does not support threaddump.
1. check this agent version is 1.6.1+
2. check cluster feature is enabled.
-----------------------------------

Actually, I used 1.8 version. And followed the configuration, but it does not work. Any suggestion?
collector:
cluster.listen.ip=0.0.0.0
cluster.listen.port=9997

web: 
cluster.connect.address=10.10.10.6:9997


Thanks,
Aaron
